src/renderer/shared/ui/Buttons/Button/common/constants.ts

```diff
@@ -0,0 +1,32 @@
+import { Pallet } from './types';
+
+export const SizeStyle: Record<'sm' | 'md', string> = {
+  sm: 'h-7 px-3 rounded-[14px]',
+  md: 'h-8 px-4 rounded-[16px]',
+};
+
+export const ButtonStyle: Record<Pallet, string> = {
+  primary:
+    'bg-button-primary-default transition-colors ' +
+    'hover:bg-button-primary-hover active:bg-button-primary-active disabled:bg-button-primary-disabled',
```

Comment:
What do you think about to use `cnTw` in such places? It will help to avoid spaces for correct concatenation